### PR TITLE
Update COR delegation

### PR DESCRIPTION
**Pull request by lfrohlich**
Deleted Dvora's COR delegation, uploaded mine, and updated team composition

### CODE REVIEW

File: docs/How-We-Work/Team-Composition.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-* Lauren Frohlich, OFA, product owner
+* Lauren Frohlich, OFA, product owner/project COR
 * Alexandra Pennington, OFA, tech lead
 
 **Raft**
 * Andrew Bottoms, Raft, tech lead
 * Jorge Gonzalez, Raft, frontend engineer
 * Aaron Beavers, Raft, backend engineer
-* Miles Reiter, Raft, user researcher
+* Miles Reiter, Raft, design lead/senior user researcher
```
Comment (lfrohlich) [inline docs/How-We-Work/Team-Composition.md:10]:
```suggestion
* Miles Reiter, Raft, design lead + senior ux/ui researcher and designer
```

File: docs/How-We-Work/Team-Composition.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 * John Willis, Raft, technical oversight support
 * Valerie Collignon, Raft, facilitator/project manager
 * Sreedevi Purayannur, Raft, senior ux/ui research and designer
```
Comment (lfrohlich) [inline docs/How-We-Work/Team-Composition.md:13]:
```suggestion
* Sreedevi Purayannur, Raft, senior ux/ui researcher and designer
```

**Comment by lfrohlich** [comment]
@ADPennington ready for your review. I'm not 100% sure i did the docs correctly :)

File: docs/How-We-Work/Team-Composition.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
 ## Primary Product Team
 **OFA**
-* Lauren Frohlich, OFA, product owner
+* Lauren Frohlich, OFA, product owner/project COR
 * Alexandra Pennington, OFA, tech lead
 
 **Raft**
 * Andrew Bottoms, Raft, tech lead
 * Jorge Gonzalez, Raft, frontend engineer
```
Comment (ADPennington) [inline docs/How-We-Work/Team-Composition.md:8]:
```suggestion
* Jorge Gonzalez, Raft, full stack engineer
```

File: docs/How-We-Work/Team-Composition.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 **OFA**
-* Lauren Frohlich, OFA, product owner
+* Lauren Frohlich, OFA, product owner/project COR
 * Alexandra Pennington, OFA, tech lead
 
 **Raft**
 * Andrew Bottoms, Raft, tech lead
-* Jorge Gonzalez, Raft, frontend engineer
+* Jorge Gonzalez, Raft, full stack engineer
 * Aaron Beavers, Raft, backend engineer
```
Comment (ADPennington) [inline docs/How-We-Work/Team-Composition.md:9]:
```suggestion
* Aaron Beavers, Raft, full stack engineer
```

**Review by ADPennington — APPROVED** [review]
lgtm 👍🏾

**Pull request merged**